docs/source/aryn_cloud/gentle_introduction.md

```diff
@@ -0,0 +1,220 @@
+## A Gentle Introduction to the Aryn Partitioning Service 
+You can use the Aryn Partitioning Service to easily chunk and extract data from complex PDFs. The Partitioning Service can extract paragraphs, tables and  images and returns detailed information about the components it has just identified in a JSON object.  The following two sections will walk you through two examples where we segment  PDF documents and extract a table and an image from those documents using the python aryn-sdk.
+
+### Extracting Tables from a PDF
+
+In [this example](https://colab.research.google.com/drive/1ZkJ9clSVVvJsNPRlWXpplfs1U9W4ZSdH?usp=sharing), we’ll use the Partitioning Service to extract the “Supplemental Income” table (shown below) from the 10k financial document  of 3M, and turn it into a pandas dataframe. 
+
+![alt text](3m_supplemental_income.png)
+
+We’ll go through the important code snippets below  to see what’s going on.  (Try out it out in  [colab](https://colab.research.google.com/drive/1ZkJ9clSVVvJsNPRlWXpplfs1U9W4ZSdH?usp=sharing)  yourself! )
+
+
+Let’s focus on the following code that makes a call to the Aryn Partitioning Service: 
+
+```
+import aryn_sdk
+from aryn_sdk.partition import partition_file, tables_to_pandas
+import pandas as pd
+from io import BytesIO
+
+## Make a call to the Aryn Partitioning Service (APS) 
+## param extract_table_structure (boolean): extract tables and their structural content. default: False
+## param use_ocr (boolean): extract text using an OCR model instead of extracting embedded text in PDF. default: False
+## returns: JSON object with elements representing information inside the PDF
+partitioned_file = partition_file(curr_file, aryn_api_key, extract_table_structure=True, use_ocr=True)
```

Comment:
what's curr_file?

Reply:
hmm for some reason i thought "file" was a keyword in python (colab kept highlighting it as such). I'll change it to "file" 

Also I didn't want to comment every single parameter either because I felt like it distracted from the actual line of code where we call "partition_file"

Reply:
Oh it's defined in the notebook? I'm just seeing a random param called curr_file and not knowing where it's defined or what kind of thing it is. Might be worth adding a 
```python
curr_file = open('my-document.pdf', 'rb')
```
in the block